src/main/scala/com/exini/dicom/streams/ParseFlow.scala

```diff
@@ -114,7 +120,7 @@ class ParseFlow private (chunkSize: Int) extends ByteStringParser[DicomPart] {
                     log.warning(s"File meta information uses big-endian encoding")
                   InFmiHeader(FmiHeaderState(None, info.bigEndian, info.explicitVR, info.hasFmi, 0, None))
                 } else
-                  InDatasetHeader(DatasetHeaderState(0, info.bigEndian, info.explicitVR))
+                  InDatasetHeader(DatasetHeaderState(0, maySwitchTs = true, info.bigEndian, info.explicitVR))
```

Comment:
By setting maySwitchTs to true here we are able to parse data that claims to be implicit VR but is really explicit VR